src/v/kafka/requests/fetch_request.cc

```diff
@@ -315,7 +338,9 @@ read_from_ntp(op_context& octx, model::ntp ntp, fetch_config config) {
      * the tp in the metadata cache so that this condition is unlikely
      * to pass.
      */
-    auto shard = octx.rctx.shards().shard_for(ntp);
+    const auto mntpv = model::materialized_ntp(std::move(ntp));
```

Comment:
nit: can we name this variable `m_ntp` ? why there is a `v` at the end ?

Reply:
lets take care of this in a separate patch